DTS-STN/Service-Canada-Labs
# Modify storybook webpack config to fix svg loader

# [Storybook not bringing in SVG resources](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities)

This PR fixes an issue where SVGs referenced in storybook components were not rendering. Modifications include:

- Adding staticDirs prop to storybook config
- Use @svgr/webpack as recommended in the [Storybook with Next](https://github.com/storybookjs/storybook/tree/next/code/frameworks/nextjs#custom-webpack-config) docs
- Updated related dependencies

## Test Instructions

1. yarn storybook
2. Navigate to Molecules/CallToAction
3. See that speech bubble svg renders
